aea/agent_loop.py

```diff
@@ -409,28 +284,29 @@ async def _task_process_new_behaviours(self) -> None:
 class SyncAgentLoop(BaseAgentLoop):
     """Synchronous agent loop."""
 
-    def __init__(self, agent: "Agent") -> None:
+    def __init__(self, agent: "Agent", loop: AbstractEventLoop = None):
         """
         Init agent loop.
 
-        :param agent: agent or AEA instance.
+        :param agent: AEA instance
+        :param loop: asyncio loop to use. optional
         """
-        super().__init__(agent)
-        self.is_running = False
+        super().__init__(agent=agent, loop=loop)
+        self._agent: "AEA" = self._agent
+        asyncio.set_event_loop(self._loop)
 
-    def start(self) -> None:
-        """Start agent loop."""
-        self.is_running = True
```

Comment:
what happened to `self.is_running`?

Reply:
it moved  to is_running property depends on state